### ISSUE TITLE
use glow backend instead of wgpu

### ISSUE THREAD

**Issue by edouardparis** [revault/revault-gui#60]
wgpu force user to install vulkan driver.
glow may be a solution: https://github.com/hecrj/iced/issues/613#issuecomment-725550260
We will lost svg and qr_code features but we may be able to replace them with png and iced Image

**Comment by darosior** [comment]
As discussed IRL with Edouard, dropping SVG support is fine as we only use it for the Revault logo at the very top left. For the Qrcode i don't know, it seems suboptimal  to use a PNG.

**Comment by darosior** [comment]
What's the state of this? Is it still doable as much as it was 3 months ago? Are we sure it will get rid of Vulkan? Is there anyother tradeoff?

**Comment by edouardparis** [comment]
More information: https://github.com/hecrj/iced/issues/935

**Comment by darosior** [comment]
Yep know about this issue. Looks like it's very doable but my question was especially regarding our use of the library, and if we introduced new incompatible uses in the past 3 months.

If we didn't it should be a priority to get rid of the Vulkan dependency. Please tell me if this opinion is missing some info.

-------- Original Message --------
